app/cafe.py

```diff
@@ -0,0 +1,20 @@
+import datetime
+from app.errors import (
+    NotVaccinatedError,
+    OutdatedVaccineError,
+    NotWearingMaskError
+)
+
+
+class Cafe:
+    def __init__(self, name: str) -> None:
+        self.name = name
+
+    def visit_cafe(self, visitor: dict) -> str:
+        if not visitor.get("vaccine"):
+            raise NotVaccinatedError("Visitor should be vaccinated.")
+        if not visitor["vaccine"]["expiration_date"] >= datetime.date.today():

```

Comment:
better to use get() in all places where you access a key

Reply:
> better to use get() in all places where you access a key

exactly, thanks!